packages/app-accounts/src/ChangePass.tsx

```diff
@@ -41,11 +42,14 @@ class ChangePass extends React.PureComponent<Props, State> {
   }
 
   render () {
+    const { className } = this.props;
+
     return (
       <Modal
-        size='tiny'
+        className={classes('accounts--ChangePass-Modal', className)}
```

Comment:
(nitpick) Since we never override, we don't need classes (rather let's keep it to ui-* and where exactly needed - we have been adding this all-over in the past and in 99.9% of the cases it was just extra fluff with no benefit. When needed, can add - there are reason we may want to)